fedimint-api/src/encoding/tbs.rs

```diff
@@ -35,6 +37,30 @@ impl_external_encode_bls!(tbs::BlindedSignatureShare, tbs::MessagePoint, 48);
 impl_external_encode_bls!(tbs::BlindedSignature, tbs::MessagePoint, 48);
 impl_external_encode_bls!(tbs::Signature, tbs::MessagePoint, 48);
 
+impl Encodable for threshold_crypto::PublicKeySet {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        SerdeEncodable(self.clone()).consensus_encode(writer)
```

Comment:
Using `SerdeEncodable` in consensus is to be avoided, because it pulls in bincode we we don't generally trust as much. Giving at a blanket impl could invite accidental use.

Possibly you should use `SerdeEncodable` directly on the field you need to encode, and we should think if it doesn't affect consensus.

Reply:
Ping @elsirion for second opinion.

It might be a good reason to impl a to/from bytes function for `threshold_crypto` types directly, and then use them here, instead of `SerdeEncodable`.

Reply:
Kinda agree, can we just impl a `to_bytes` fn in our fork of `threshold_crypto`? Might be easiest.

Reply:
Let's mimic the naming that secp256k1 use: https://docs.rs/secp256k1/latest/secp256k1/struct.SecretKey.html#method.secret_bytes @jkitman 

(sorry for the detour! :D )

Reply:
I will expose a method to get the coefficients then encode them, similar to what we do in the TBS crate.